db/Models.py

```diff
@@ -327,6 +332,12 @@ def update_state(self, packet):
                 self.join_request_counter += 1
                 self.is_otaa = True
             self.last_packet_id = packet.id
+            if packet.m_type in ["UnconfirmedDataUp", "ConfirmedDataUp", "JoinRequest"]:
+                packets_list = json.loads(self.last_packets_list)
+                packets_list.append(packet.id)
+                if len(packets_list) > self.MAX_PACKETS_LIST_SIZE:
```

Comment:
I understand it is probably not necessary, but, can we change the if for a while?

Reply:
I see your point, done!